docs/templating-language.md

```diff
@@ -775,12 +775,16 @@ Query [Consul][consul] for services based on their health.
 
 The `<TAG>` attribute is optional; if omitted, all nodes will be queried.
 
-The `<QUERY>` attribute is optional; if omitted, the `default` Consul namespace, `default` partition will be queried. `<QUERY>` can be used to set the Consul [namespace](https://developer.hashicorp.com/consul/api-docs/health#ns-2), partition, or [peer](https://developer.hashicorp.com/consul/api-docs/health#peer). `<QUERY>` accepts a url query-parameter format, e.g.:
```

Comment:
Should we document what will be returned with sameness group and how it will behave with failures and updates to the config entry?

Reply:
I've updated this in 7002f6235037a0c136fbff7e4594349fbbbad4c6.